hooks/pre-commit

```diff
@@ -3,14 +3,23 @@
 jsfiles=$(git diff --cached --name-only --diff-filter=ACM "*.js" "*.jsx" "*.ts" "*.tsx" | tr '\n' ' ')
 [ -z "$jsfiles" ] && exit 0
 
-# Check if any of the files are ugly
+# Check if any of the files are ugly or contain a console.log call
 uglyFiles=()
+consoleFiles=""
 for jsfile in $jsfiles; do
```

Comment:
Is bash clever enough to handle files with spaces in here? Does git diff quote them automatically?

Reply:
I don't think so. I should probably remove the `tr` here: https://github.com/cmcaine/tridactyl/pull/561/commits/dbdba26f9bda8ce0f7d423e0cd1d4fadfe5efe0f#diff-5a2bdf4dec697a53c4a4251a27481031R3 and change the IFS to '\n'. I'ill try that later today.